### ISSUE TITLE
Contradictory statements for Fortran binding

### ISSUE THREAD

**Issue by qbit19** [cp2k/libint-cp2k#5]
When I want to download libint tarball I am directed to page:
https://github.com/cp2k/libint-cp2k/releases/tag/v2.6.0
where it is written:

libint v2.6.0 configured for use with CP2K

Note: **cmake build of Fortran interface is broken in this release, please use autoconf to build this libint version**.

However, on the main build page "Build instructions" I am adviced to use the wiki:
https://github.com/evaleev/libint/wiki#compiling-libint-library
where it is written:
As of version 2.6.0 the Autoconf build is deprecated; the exported libint library should be configured with CMake and built with any CMake-supported generator, e.g. Ninja and GNU Make. **Only CMake-based build will be discussed**.

In summary, what is this NONSENSE:

cmake build of Fortran interface is broken in this release <-> Only CMake-based build will be discussed

please use autoconf to build this libint version <-> As of version 2.6.0 the Autoconf build is deprecated; 

Are you guys reading what you are writing ??

**Comment by dev-zero** [comment]
You are confusing the libint project with our project here, which is just a service to publicly document the default cp2k libint settings, in form of code. The `Jenkinsfile` shows how to generate the tarballs we ship on the cp2k.org mirror of source files. While the toolchain in CP2K [can be used as an example on how to build it](https://github.com/cp2k/cp2k/blob/master/tools/toolchain/scripts/stage3/install_libint.sh).

**Comment by alazzaro** [comment]
Thanks for spotting this inconsistency between the CP2K readme (which was dated 2 years ago) and the recent libint wiki page (which was updated 4 months ago).

The suggested way to install dependencies with CP2K is the CP2K toolchain. Alternatively, you can use building tools such as SPACK or Easybuild.
 
> Are you guys reading what you are writing ??

Besides the fact that this is a bit offending, you are always welcome to open a PR and update the README. As stated above, the libint wiki page changed in the meantime (we don't control libint page). With the recent commit #6 I've removed this link.
